docs/generators/scala-akka-http-server.md

```diff
@@ -33,6 +33,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |legacyDiscriminatorBehavior|Set to false for generators with better support for discriminators. (Python, Java, Go, PowerShell, C# have this enabled by default).|<dl><dt>**true**</dt><dd>The mapping in the discriminator includes descendent schemas that allOf inherit from self and the discriminator mapping schemas in the OAS document.</dd><dt>**false**</dt><dd>The mapping in the discriminator includes any descendent schemas that allOf inherit from self, any oneOf schemas, any anyOf schemas, any x-discriminator-values, and the discriminator mapping schemas in the OAS document AND Codegen validates that oneOf and anyOf schemas contain the required discriminator and throws an error if the discriminator is missing.</dd></dl>|true|
 |modelPackage|package for generated models| |null|
 |modelPropertyNaming|Naming convention for the property: 'camelCase', 'PascalCase', 'snake_case' and 'original', which keeps the original name| |camelCase|
+|pekkoHttpVersion|The version of pekko-http| |1.0.0|
```

Comment:
thanks for the PR which looks good to me

one minor suggestion: what about defaulting to the latest stable version 1.1.0 instead?

https://mvnrepository.com/artifact/org.apache.pekko/pekko-http_3/

Reply:
I was thinking about that as well but wasn't sure if it would be a problem for anyone. But if you think it's fine I can include that change as well, think it makes sense!

Reply:
Pushed a fix.

Reply:
One can rollback the dependency to a particular version using customized templates (e.g. -t via CLI) so let's move forward with the latest stable version which is what we've been doing in other generators.